src/main/java/com/palantir/gradle/versions/GetVersionPlugin.java

```diff
@@ -105,30 +105,21 @@ private static String getVersion(Project project, String group, String name, Con
         }
 
         if (list.size() > 1) {
-            throw new GradleException(String.format("Multiple modules matching '%s:%s' in %s: %s",
-                    group,
-                    name,
-                    configuration,
-                    list));
+            throw new GradleException(String.format(
+                    "Multiple modules matching '%s:%s' in %s: %s", group, name, configuration, list));
         }
 
         return Optional.of(Iterables.getOnlyElement(list).getVersion());
     }
 
     private static GradleException notFound(String group, String name, Configuration configuration) {
-        String actual = configuration.getIncoming()
-                .getResolutionResult()
-                .getAllComponents()
-                .stream()
+        String actual = configuration.getIncoming().getResolutionResult().getAllComponents().stream()
                 .map(ResolvedComponentResult::getModuleVersion)
                 .map(mvi -> String.format("\t- %s:%s:%s", mvi.getGroup(), mvi.getName(), mvi.getVersion()))
                 .collect(Collectors.joining("\n"));
         return new GradleException(String.format(
                 "Unable to find '%s:%s' in %s. This may happen if you specify the version in versions.props but do not"
                         + " have a dependency in the configuration. The configuration contained:\n%s",
-                group,
-                name,
-                configuration,
-                actual));
+                group, name, configuration, actual));
```

Comment:
what's the heuristic that pushes this into one line?

Reply:
GJF handles `String.format` arguments magically.
It decouples the arguments from the format string, so the format string can be broken (split into lines) separately from the arguments.
Because the format string comes first, it gets broken (as it won't fit on one line) and that ensures another newline _after_ it, but then the arguments themselves fit on one line so they are left as is.